README.md

```diff
@@ -22,6 +22,8 @@ It is possible to only include data in reports based on specific conditions (Pup
 
 Enabling this module is as simple as classifying your Puppet Servers with spluk_hec and setting the Splunk HEC URL along with the token provided by Splunk. This module sends data to Splunk by modifying your report processor settings and indirector routes.yaml.
```

Comment:
Nit: typo - `spluk_hec`

Reply:
In order to start sending reports I think you also have to set `enable_reports` to true.